Spec.md

```diff
@@ -211,6 +212,101 @@ status:
 
 ```
 
+#### ExternalSecret - Azure Objects
+
+The 5 semantics below (3 for indovidual secrets, 2 for KeyVault) refect the
+[Azure API](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault) capability:
+
+```yaml
+  data:
+    # The name of the managed Kubernetes Secret
+    - secretKey: a-simple-secret
+      # The reference to the Azure Secret value
+      remoteRef:
+        # The Azure Secret object name
+        key: a-simple-secret-azure-name
+
+    # The name of the managed Kubernetes Secret
+    - secretKey: a-versioned-secret
+      # The reference to the Azure Secret value
+      remoteRef:
+        # The Azure Secret object name
+        key: a-versioned-secret-azure-name
+        # The Secret version (optional: default is latest)
+        version: a-versioned-secret-azure-version
+
+    # The name of the managed Kubernetes Secret
+    - secretKey: a-secret-tag
+      # The reference to the Azure Secret tag
+      remoteRef:
+        # The Azure Secret object name
+        key: a-tagged-secret-azure-name
+        # The (optional) tag name to fetch the value from as secret value
+        property: a-tagged-secret-tag-name
+
+    # A managed Kubernetes TLS-typed Secret from a TLS CERT-bundle in Azure (proposal 1, with new field "type")
+    - secretKey: tls.key
+      # The reference to the certificate bundle in Azure Secret
+      remoteRef:
+        # The Azure Secret object name
+        key: a-tls-secret-azure-name
+        # How much of the CERT bundle we want to get: one of "key" | "cert" | "bundle" (default)
+        type: key
+    - secretKey: tls.cert
+      # The reference to the certificate bundle in Azure Secret
+      remoteRef:
+        # The Azure Secret object name
+        key: a-tls-secret-azure-name
+        # How much of the CERT bundle we want to get: one of "key" | "cert" | "bundle" (default)
+        type: cert
+
+    # A managed Kubernetes TLS-typed Secret from a TLS CERT-bundle in Azure (proposal 2, with "/qualifier")
+    - secretKey: tls.key
+      # The reference to the certificate bundle in Azure Secret
+      remoteRef:
+        # The Azure Secret object name
+        key: a-tls-secret-azure-name/tls.key
+    - secretKey: tls.cert
+      # The reference to the certificate bundle in Azure Secret
+      remoteRef:
+        # The Azure Secret object name
+        key: a-tls-secret-azure-name/tls.cert
+
+dataFrom:
```

Comment:
Looks like indentation is off for this line 
```suggestion
  dataFrom:
```
not in the loop enough for other comments but noticed this 😄